organization/_posts/2021/2021-03-18-coordination.md

```diff
@@ -0,0 +1,153 @@
+
+Andrew: wanted to define a timescale on which feedback was given - early enough to be able to take requirements and statements back to FAs, viz. 4-6 weeks... (timescale is driven by US funding and a computing CDR that is due in June).
+
+- To crosscheck the requirements or to check how much existing software matches those (fixed) requirements? Would not be able to do both on a short timescale.
+
+Andrew: check on requirements should be a quick part; real question is how do those requirements fit into the HEP ecosystem?
+
+- We would try to leverage the broadest expertise possible, of people willing to help
+- Are the facility boundary conditions clear? GPUs or not? Dedicated or shared resources? Special facilities like analysis facilities?
+
+Andrew: there is a description of some of this and we do anticipate being quite free to decide how to use these resources; data sizes and form of a neutrino analysis are well known. Some feedback on this aspect would also be useful, when people come with different experience.
+
+- That would require some different skills (don't want to have an unmanageable group)
+
+Andrew: HSF is an avenue to different expertise that we would like to take advantage of.
+
+- Have a kick-off talk in a frameworks group meeting?
+    - Yes, this is a good idea
+    - Chris: agree, but we do want the HSF to assemble a panel of suitable experts
+
+Benedikt, Chris, Kyle and Attila will discuss then how to proceed.
+
+## Working Group Updates
+
+### Data Analysis
+
+- Work has started on metadata document. After the skeleton document has been created will ask metadata meeting contributors to help populate it
+- Organisation of next meeting series on Analysis frameworks has begun (approx. timeline mid-April). Want to focus on the non-LHC experiments, which will be split into developing and established experiment categories. Contacting/investigating potential speakers, please let us know if you have names in mind.
+- Current (tentative) list:
+    - (proto)DUNE
+    - MicroBooNE
+    - BNL (RHIC/EIC)
+    - JLab
+    - FAIR
+    - LZ
+    - ISOLDE?
+
+### Detector Simulation
+
+- Next two meetings scheduled: <https://indico.cern.ch/category/10916/>
+    - 12th April: [Using GPUs in detector simulations](https://indico.cern.ch/event/1019940/). Have updates on AdePT, Celeritas and Opticks/Geant4 examples. See this as opportunity for these projects to report on computing model/progress/plans, _and_ to solicit initial feedback from simulation _users_ on progress on these topics. We will advertise through the usual lists, but wider advertisement very welcome, especially outside of LHC experiments!
+    - 10th May: [Simulating solid-state microphysics with Geant4 and G4CMP](https://indico.cern.ch/event/1016632/)
+
+### Reconstruction and Software Triggers
+
+- Kick-off meeting evaluating the outcome of the survey held on Wednesday: https://indico.cern.ch/event/1011873/
```

Comment:
I can't remember now if URLs should be in between <...> or not? This one (and others) isn't, but 4 lines above the <...> syntax is used. Just act as/if necessary :-).

Reply:
Oh, yes, one does need the <>s - didn't spot this one, will fix.